RecoEgamma/EgammaElectronProducers/python/gsfElectronSequence_cff.py

```diff
@@ -12,6 +12,10 @@
 _gsfEcalDrivenElectronSequenceFromMultiCl = gsfEcalDrivenElectronSequence.copy()
 _gsfEcalDrivenElectronSequenceFromMultiCl += cms.Sequence(ecalDrivenGsfElectronCoresFromMultiCl*ecalDrivenGsfElectronsFromMultiCl)
 
+# Low Pt Electrons
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronSequence_cff import *
+gsfEcalDrivenElectronSequence *= lowPtGsfElectronSequence
```

Comment:
@Sam-Harper I push the LowPtEle sequence in here ... which is included [here](https://github.com/bainbrid/cmssw/blob/LowPtElectrons_prod/RecoEgamma/Configuration/python/RecoEgamma_cff.py#L25) ... which is included [here](https://github.com/bainbrid/cmssw/blob/LowPtElectrons_prod/Configuration/StandardSequences/python/Reconstruction_cff.py#L113). Is this the best place to do this? Also, I need to edit the [EventContent](https://github.com/bainbrid/cmssw/blob/LowPtElectrons_prod/Configuration/EventContent/python/EventContent_cff.py) to make the new collections persistent. I've done this on other branches, but yet to do so on this PR as I'm unsure as to whether this is allowed or not ...?